compatibility_server/compatibility_checker_server.py

```diff
@@ -124,24 +113,17 @@ def _check(self, start_response, python_version, packages):
             ]
         python_command = self._python_version_to_interpreter[python_version]
 
-        if self._install_once:
-            self._shutdown()
-
         try:
             pip_result = pip_checker.check(
-                [python_command, '-m', 'pip'], packages, clean=self._clean)
-        except pip_checker.PipError as pip_error:
+                [python_command, '-m', 'pip'], packages)
```

Comment:
I would still use an enum and not pass the whole command.

Reply:
I changed to use a dict mapping the python version to the pip command. Seems like using an enum will add more conversions in here? Like `'2' -> PythonVersionEnum.PYTHON2 -> ['python2', '-m', 'pip']`